src/core/structures/context.ts

```diff
@@ -45,6 +50,10 @@ export class Context extends CoreContext<Message, ChatInputCommandInteraction> {
         return safeUnwrap(this.ctx.map(m => m.author).mapErr(i => i.user));
     }
 
+    public get userId(): Snowflake {
+        return this.user.id;
```

Comment:
userId is always guaranteed? is it possible to be from webhook or nah 🤔

Reply:
it's guaranteed to have some user on the Context object

Reply:
bet